arbor/communication/dry_run_context.cpp

```diff
@@ -49,25 +47,24 @@ struct dry_run_context_impl {
     void remote_ctrl_send_done() const {}
     gathered_vector<cell_gid_type>
     gather_gids(const std::vector<cell_gid_type>& local_gids) const {
-        using count_type = typename gathered_vector<cell_gid_type>::count_type;
-
-        count_type local_size = local_gids.size();
+        size_t local_size = local_gids.size();
```

Comment:
`std::size_t`?

Reply:
Yes and all other instance in that file.